openbook_notifications/serializers.py

```diff
@@ -11,15 +11,22 @@
     PostCommentReactionNotification, PostCommentUserMentionNotification, PostUserMentionNotification
 from openbook_notifications.models.post_reaction_notification import PostReactionNotification
 from openbook_notifications.serializer_fields import ParentCommentField
-from openbook_notifications.validators import notification_id_exists
 from openbook_posts.models import PostComment, PostReaction, Post, PostImage, PostVideo, PostCommentReaction, \
     PostUserMention, PostCommentUserMention
+from openbook_notifications.validators import notification_id_exists, notification_type_exists
 
 
 class ReadNotificationsSerializer(serializers.Serializer):
     max_id = serializers.IntegerField(
         required=False,
     )
+    types = serializers.ListField(
+        child=serializers.CharField(
```

Comment:
`ChoiceField` is more appropiate here, it will return available values on error messages. The list we've manually added in other places but you can also do the for loop too.

Reply:
The same applies to `GetNotificationSerializer`, I presume?